RecoTracker/LSTCore/src/alpaka/Segment.h

```diff
@@ -359,6 +379,13 @@ namespace lst {
     segmentsInGPU.outerLowerModuleIndices[idx] = outerLowerModuleIndex;
     segmentsInGPU.innerMiniDoubletAnchorHitIndices[idx] = innerMDAnchorHitIndex;
     segmentsInGPU.outerMiniDoubletAnchorHitIndices[idx] = outerMDAnchorHitIndex;
+    segmentsInGPU.innerMiniDoubletOuterHitIndices[idx] = innerMDOuterHitIndex;
+    segmentsInGPU.outerMiniDoubletOuterHitIndices[idx] = outerMDOuterHitIndex;
+
+    segmentsInGPU.logicalLayers[idx* Params_LS::kLayers] =
+        modulesInGPU.layers[innerLowerModuleIndex] + (modulesInGPU.subdets[innerLowerModuleIndex] == 4) * 6;
+    segmentsInGPU.logicalLayers[idx * Params_LS::kLayers + 1] =
+        modulesInGPU.layers[innerLowerModuleIndex] + (modulesInGPU.subdets[innerLowerModuleIndex] == 4) * 6;
```

Comment:
should these be outer?
```suggestion
        modulesInGPU.layers[outerLowerModuleIndex] + (modulesInGPU.subdets[outerLowerModuleIndex] == 4) * 6;
```

Reply:
Yes, thank you for pointing!